astroquery/wfau/core.py

```diff
@@ -0,0 +1,903 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import print_function
+
+import warnings
+import re
+import time
+from math import cos, radians
+import requests
+from bs4 import BeautifulSoup
+from io import StringIO
+
+from astropy.extern.six import BytesIO
+import astropy.units as u
+import astropy.coordinates as coord
+import astropy.io.votable as votable
+
+from ..query import QueryWithLogin
+from ..exceptions import InvalidQueryError, TimeoutError, NoResultsWarning
+from ..utils import commons
+from ..exceptions import TableParseError
+
+__all__ = ['BaseWFAUClass', 'clean_catalog']
+
+
+class BaseWFAUClass(QueryWithLogin):
```

Comment:
Is there a reason that ``async_to_sync`` is not used?

Reply:
Most likely because the original UKIDSS development preceded async_to_sync.  I'll complete that refactor if I get to it, but I want to think through it too...